editor/import/editor_scene_importer_gltf.h

```diff
@@ -279,6 +279,9 @@ class EditorSceneImporterGLTF : public EditorSceneImporter {
 
 		Map<GLTFNodeIndex, Node *> scene_nodes;
 
+		// EditorSceneImporter::ImportFlags
+		uint32_t import_flags;
```

Comment:
Why can't you use the enum directly instead of `uint32_t`? This class inherits `EditorSceneImporter`.

Reply:
No super strong reason.  By this point in the flow, the flags are already being passed around as uint32_t so I kept it that way to avoid casting back and forth.  I assume it's this way for brevity of performing bitwise ops.

It's populated from p_flags...
`EditorSceneImporterGLTF::import_scene(const String &p_path, uint32_t p_flags, int p_bake_fps, List<String> *r_missing_deps, Error *r_err)`

Reply:
Alright, if it's consistent with existing code it's good as is, no need to refactor in this PR :)